--- rc-stuff.c~	Thu Nov  4 23:54:47 1999
+++ rc-stuff.c	Tue Nov  9 13:13:29 1999
@@ -258,11 +258,11 @@
 	strcat(buf, "/.wmsetirc");
 	if ((fp = fopen(buf, "r")) == NULL)
 	{
-		strcpy(buf, "/usr/local/etc/wmsetirc");
+		strcpy(buf, "/usr/X11R6/share/wmsetirc");
 		if ((fp = fopen(buf, "r")) == NULL)
 		{
 			if (Verbose) 
-				fprintf(stderr, "configuration file not found ($HOME/.wmsetirc or /etc/wmsetirc)\n");
+				fprintf(stderr, "configuration file not found ($HOME/.wmsetirc or /usr/X11R6/share/wmsetirc)\n");
 			return NULL;
 		}
 		else
