diff -urN wmseti-1.0.3/README wmseti-1.0.3.rc.patch/README
--- wmseti-1.0.3/README	Sun Mar  3 16:12:38 2002
+++ wmseti-1.0.3.rc.patch/README	Sun Dec 28 21:18:31 2003
@@ -43,10 +43,10 @@
 be used to go up and down the directories in either display.
 
 The configuration file as been slightly changed from the original
-version. See "wmsetirc" for an example, or copy it to "~/.wmsetirc"
-and change it to your needs. The default display settings are what
-I use. The configuration covers seti@home directories, program
-command line, statistics display, refresh and update rates.
+version. See "/etc/wmsetirc" and change it to your needs. The default
+display settings are what I use. The configuration covers seti@home
+directories, program command line, statistics display, refresh and
+update rates.
 
 The timeleft statistic now shows whether an upload or download
 to the SETI@home server is required.
diff -urN wmseti-1.0.3/src/wmseti.c wmseti-1.0.3.rc.patch/src/wmseti.c
--- wmseti-1.0.3/src/wmseti.c	Sat Dec 21 01:13:40 2002
+++ wmseti-1.0.3.rc.patch/src/wmseti.c	Sun Dec 28 21:16:12 2003
@@ -368,8 +368,8 @@
 	}
 	optind = 0;
 
-	strcpy( buf,  "~/.wmsetirc" );
-	expand_dir( buf );
+	strcpy( buf,  "/etc/wmsetirc" );
+/*	expand_dir( buf );	*/
 
 	if ( fp = fopen( buf, "r" ) ) {
 		while ( fgets( buf, sizeof(buf), fp ) ) {
